jax/_src/nn/initializers.py

```diff
@@ -48,7 +48,67 @@ def _compute_fans(shape: core.NamedShape, in_axis=-2, out_axis=-1):
   fan_out = shape[out_axis] * receptive_field_size
   return fan_in, fan_out
 
+def _get_real_dtype(dtype):
```

Comment:
Is there already an util function in the jax codebase to do this?

Reply:
It's definitely done elsewhere in the package, but perhaps a one-liner is better than a utility routine? e.g. https://github.com/google/jax/blob/b6e25fa00c31c12cc4d302e80a44baf6fee3327d/jax/_src/random.py#L677

Reply:
Here's another version: https://github.com/google/jax/blob/97a5719fcb40af7231b5f803f965063538282f8e/jax/_src/lax/fft.py#L63

Reply:
Ok, let me use a one-liner